cdap-operational-stats-core/src/test/java/co/cask/cdap/operations/cdap/CDAPOperationalStatsTest.java

```diff
@@ -91,7 +91,9 @@ public void test() throws Exception {
     Assert.assertEquals(1, transactions.getNumInProgressTransactions());
     Assert.assertEquals(1, transactions.getNumInvalidTransactions());
     Assert.assertEquals(1, transactions.getNumCommittingChangeSets());
-    Assert.assertEquals(0, transactions.getNumCommittedChangeSets());
+    // the entites.collect calls appLifecycleService.getApps this fetches the owner principal of the deployed which
```

Comment:
this test creates transactions in the setup, and then asserts them here. It seems bad that something other than the setup impacted this verification.

I think even for making this more foolproof, these verifications should be changed to not assert on equality, but instead use ``>=``